docs/Changelog.md

```diff
@@ -16,7 +16,11 @@ sending a mail to <afl-users+subscribe@googlegroups.com>.
        exactly one main node is present and warn otherwise
      - If no main node is present at a sync one secondary node automatically becomes
        a temporary main node until a real main nodes shows up
+     - switched murmur2 hashing and random() for xxh3 and xoshiro256**, giving up to 5.5% speed
+       increase
+     - Resizing the window does not crash afl-fuzz anymore
      - fix/update to MOpt (thanks to arnow117)
+     - added MOpt dictionary support from repo
```

Comment:
note: the dictionary support seems to have an issue, the author removed it today again. I will keep an eye on it.